content/_index.en.html

```diff
@@ -222,4 +237,77 @@ <h3>Medium</h3>
 		scrollTop: position
 	} /* speed */ );
 });
+
+$(".video-eco").magnificPopup({
+  type: "iframe",
+
+  iframe: {
+    markup:
+      '<div class="mfp-iframe-scaler">' +
+      '<div class="mfp-close"></div>' +
+      '<iframe class="mfp-iframe" frameborder="0" allowfullscreen></iframe>' +
+      '<div class="mfp-title">Some caption</div>' +
+      "</div>"
+  },
+  callbacks: {
+    markupParse: function(template, values, item) {
+      values.title = item.el.attr("title");
+    }
+  }
+});
+
+
 </script>
+
+
+<style>
+.container-eco {
```

Comment:
Move this CSS into the appropriate scss file please, into the homepage's scss file or `nav.scss` if we don't have a file for the homepage.

Reply:
done